components/pill-container/private/selected-listbox.jsx

```diff
@@ -251,6 +251,7 @@ const SelectedListBox = (props) =>
 								icon={icon}
 								labels={{
 									label: option.label,
+									title: option.title || option.label,
```

Comment:
I'd say this should be a `??` so that it's possible to force an empty title instead of defaulting to the label.